.travis.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       name: "Build and test root: nightly"
       rust: nightly
       script:
+        - rustup default nightly-2020-09-21 # remove this line when current external bug is fixed
```

Comment:
That's not how I meant^^ I was thinking more like https://github.com/SergioBenitez/Rocket/pull/257/files but adding as a new version so we can continue to see the build failure with the most recent nightly, but as an allowed failure.